src/main/java/com/google/alpollo/SongDataBase.java

```diff
@@ -0,0 +1,13 @@
+package com.google.alpollo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SongDataBase {
+    public static List<Song> topSongs() {
+        List<Song> songs = new ArrayList<>();
+        songs.add(new Song("The Weeknd", "Blinding Lights", "After Hours"));
```

Comment:
Please add an inline comment saying something like "we manually populate the top songs for now. TODO: automate how we populate the database"